config.json

```diff
@@ -1,4 +1,6 @@
 {
+  "warn_on_short_names": 3,
```

Comment:
how often does this happen?  I would assume we're going to see zillions of warnings...

Reply:
769,715 times in 2023jul9, and 1,857,487 times in the current Babel... so yeah, probably too often to be useful :(. I'll close this PR and make a new one just to fix the [loophole in synonym uniquification](https://github.com/TranslatorSRI/Babel/pull/165/commits/54cfcdcecac3d5a60d7aedaa442becead40d06b4).